.deprecated/collection_processor.py

```diff
@@ -119,23 +118,20 @@ def create_stac_collection(self, collection_id, collection_name, collection_desc
         """
 
         sp_extent = pystac.SpatialExtent([[-140.99778, 41.6751050889, -52.6480987209, 83.23324]])
-        capture_date = datetime.datetime.strptime('2015-10-22', '%Y-%m-%d')
-        end_capture_date = datetime.datetime.strptime('2100-10-22', '%Y-%m-%d')
+        capture_date = datetime.datetime.strptime("2015-10-22", "%Y-%m-%d")
```

Comment:
Maybe add a comment explaining why these dates where chosen.

Reply:
This is deprecated. Original code using a trial dataset with specific date ranges.